tests/inventory/test_setup_views.py

```diff
@@ -229,11 +229,19 @@ def test_update_meta_business_unit_api_enrollment_update(self):
         )
         name = get_random_string(12)
         meta_business_unit = MetaBusinessUnit.objects.create(name=name)
+        self.assertFalse(meta_business_unit.api_enrollment_enabled())
         response = self.client.get(reverse("inventory:update_mbu", args=(meta_business_unit.pk,)))
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, "inventory/edit_mbu.html")
         self.assertNotContains(response, 'disabled id="id_api_enrollment" checked')
-        meta_business_unit.create_enrollment_business_unit()
+        response = self.client.post(
```

Comment:
You changed the scope of this test quite a bit to accomodate testing for the change of the api enrollment. It would have been easier to add this to the update test above, that also includes the Audit event and keep this test about the GET methods and the state of the checkbox.

Reply:
Please, next time be more specific naming the test itself. There's no unique `update test`.